app/scrapers/youtube.js

```
@@ -0,0 +1,521 @@
+  // Manipulate the script content to make it executable outside a browser context
+  // and extract the necessary variable. This is a fragile approach.
+  evalHtmlCode = evalHtmlCode.replace(/var _\w+=\w+;/, ""); // Remove variable declarations that might conflict
+  evalHtmlCode = evalHtmlCode.replace(
+    /document\.getElementById\("ytmp3"\)\.src/,
+    "`https://ytmp3.cc`",
+  ); // Replace DOM access with string
+  let tempEvalResult;
+  try {
+    // WARNING: Using eval() can be a security risk. It's used here to execute
+    // dynamic JavaScript from the target website as per original implementation.
+    tempEvalResult = eval(
```

Biome (1.9.4)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)